app/templates/album.html

```diff
@@ -1,23 +1,23 @@
 <main class="album-view container narrow">
     <section class="clearfix">
             <div class="column half">
-                <img src="/assets/images/album_covers/01.png" class="album-cover-art">
+                <img src="{{ album.albumData.albumArtUrl }}" class="album-cover-art">
             </div>
             <div class="album-view-details column half">
-                <h2 class="album-view-title">The Colors</h2>
-                <h3 class="album-view-artist">Pablo Picasso</h3>
-                <h5 class="album-view-release-info">1909 Spanish Records</h5>
+                <h2 class="album-view-title">{{ album.albumData.title}}</h2>
+                <h3 class="album-view-artist">{{ album.albumData.artist}}</h3>
+                <h5 class="album-view-release-info">{{ album.albumData.year + " " + album.albumData.label }}</h5>
```

Comment:
One question on this approach...why not have the controller enrich the `albumData` model to include an `releaseInfo` property that does this string concatenation? that way the view isnt performing this kind of thing?